.github/actions/export/action.yml

```diff
@@ -32,23 +32,10 @@ outputs:
 runs:
   using: 'composite'
   steps:
-    - name: Setup Go
-      uses: actions/setup-go@v5
+    - name: Install Descope CLI
+      uses: descope/descopecli/.github/actions/install@main
```

Comment:
cool 👍 are you sure we need to point to main? worth checking if it will take current branch if not specified (which imo is better)

Reply:
We need a tag reference from what I've seen. This is only needed because the actions are in the same repo. If we move them to some other repo at some point we'll change this is a particular version tag.